### PR TITLE
Live 145 allow anonymous user to join the stream

### PR DESCRIPTION
**Pull request by Karolk99**
Don't worry about types, there is another pull_request with needed type changes.

### CODE REVIEW

File: lib/membrane_live_web/controllers/webinar_controller.ex [hunk 1]
```diff
@@ -59,6 +59,23 @@ defmodule MembraneLiveWeb.WebinarController do
...
+         true <- callback == (&show_callback/2) do
+      callback.(conn, Map.put(params, "webinar_db", webinar))
+    else
+      {:error, :no_webinar} ->
+        %{error: :not_found, message: "Webinar with uuid #{uuid} could not be found"}
+
+      false ->
+        %{error: :forbidden, message: "Unauthenticated user does not have access to this action."}
+    end
+  end
```
Comment (Rados13) [inline lib/membrane_live_web/controllers/webinar_controller.ex:77]: We should add tests for these edge cases.

**Review by Rados13 — CHANGES_REQUESTED** [review]
There are merge conflicts to solve.

**Review by Rados13 — APPROVED** [review]
